.chloggen/resource-to-telemetry-exclude-option.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: resourcetotelemetry
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: "Adding an option to exclude some attributes during converting to labels"
+
+# One or more tracking issues related to the change
+issues: [N/A]
```

Comment:
This is not a valid value, you can put 14339 here (the PR number)

Reply:
fixed